error_test.go

```diff
@@ -97,6 +98,26 @@ type consumingFinder interface {
 	Find(got string) (rest string, ok bool)
 }
 
+func runFinders(original string, finders []consumingFinder) error {
```

Comment:
Just to confirm my understanding, runFinders was created so that you could share the old
```
for _, f := range finders {
```
logic among the "single line" and "multi line" tests.

Reply:
Yep